week-4/schedule.md

```diff
@@ -0,0 +1,66 @@
+# Testing Week Schedule
+
+## Day 1
+
+| Time    | Activity                                    | Learning outcomes                      |
+| ------- | ------------------------------------------- | -------------------------------------- |
+| 09:45   | Intro reading                               | Web servers                            |
+| 10:00   | Intro presentation                          |                                        |
+| 10:30   | [Node intro](node-intro) tbc                | HTTP server                            |
+| 11:00   | [Node Girls workshop](node-girls)           | HTTP server, POST requests, HTML forms |
+| _13:00_ | _Lunch_                                     |                                        |
+| 14:00   | [Node Girls workshop](node-girls) continued | Testing                                |
+| 16:00   | Tech for Better                             |                                        |
+
+[node-intro]: tbc
+[node-girls]: https://github.com/node-girls/node-workshop/
+
+## Day 2
+
+| Time    | Activity                           | Learning outcomes    |
+| ------- | ---------------------------------- | -------------------- |
+| 09:45   | Intro reading                      |                      |
+| 10:00   | [Design burst: navigation](db-nav) |                      |
+| 10:30   | Navigation workshop tbc            | Hover & focus styles |
+| 11:00   | [TDD Node Server](node-tdd)        | Tape, TDD            |
+| _13:00_ | _Lunch_                            |                      |
+| 14:00   | Project intro                      |                      |
+| 14:30   | Technical spikes                   |                      |
+| 16:30   | Spike presentation prep            |                      |
+| 17:00   | Spike presentations                |                      |
+
+[db-nav]: https://docs.google.com/presentation/d/1Qus1BOmpF-3rMBiulsyUf0YVPgubFHzmH9tcu6wrc9A/edit#slide=id.g26a95a14fb_0_0
+[node-tdd]: https://github.com/foundersandcoders/ws-tdd-node-server
+
+## Day 3
+
+| Time    | Activity                  | Learning outcomes        |
+| ------- | ------------------------- | ------------------------ |
+| 09:45   | Intro reading             |                          |
+| 10:00   | [Node fetch MC](fetch-mc) | Network requests in Node |
```

Comment:
Matt Davis made a fetch request workshop (forked from the original XMLHTTPRequest one), we can use this. It has a main challenge that takes 45-60mins, and a stretch challenge for teams who finish early. I've just added it to our workshops spreadsheet 👍 

Link: https://github.com/m4v15/ws-github-api

Reply:
Sorry for the confusion, I was referring to the [Request Module](https://github.com/foundersandcoders/mc-request-module-workshop/) one that Sam revamped recently. The students have to use Node's `http` to make requests from the backend. I opened an issue on that suggesting we change it from callback to promise-based, since the students will be used to using `fetch` in the frontend in the new curriculum.